.doc_gen/metadata/transcribe-medical_metadata.yaml

```diff
@@ -0,0 +1,61 @@
+# zexi 0.4.0
+transcribe_StartMedicalTranscriptionJob:
+  title: Start an &TSCM; transcription job using an &AWS; SDK
+  title_abbrev: Start a medical transcription job
+  synopsis: start an &TSCM; transcription job.
+  category:
+  languages:
+    JavaScript:
+      versions:
+        - sdk_version: 3
+          github: javascriptv3/example_code/transcribe
+          sdkguide: sdk-for-javascript/v3/developer-guide/transcribe-medical-examples-section.html#transcribe-start-medical-transcription
+          excerpts:
+            - description: Create the client.
+              snippet_tags:
+                - transcribe.JavaScript.createclientv3
+            - description: Start a medical transcription job.
+              snippet_tags:
+                - transcribe.JavaScript.jobs.createMedicalJobV3
+  services:
+    transcribe: {StartMedicalTranscriptionJob}
+transcribe_DeleteMedicalTranscriptionJob:
+  title: Delete a &TSCM; transcription job using an &AWS; SDK
+  title_abbrev: Delete a medical transcription job
+  synopsis: delete an &TSCM; transcription job.
+  category:
+  languages:
+    JavaScript:
+      versions:
+        - sdk_version: 3
+          github: javascriptv3/example_code/transcribe
+          sdkguide: sdk-for-javascript/v3/developer-guide/transcribe-medical-examples-section.html#transcribe-delete-medical-job
+          excerpts:
+            - description: Create the client.
+              snippet_tags:
+                - transcribe.JavaScript.createclientv3
+            - description: Delete a medical transcription job.
+              snippet_tags:
+                - transcribe.JavaScript.jobs.deleteMedicalJobV3
+  services:
+    transcribe: {DeleteMedicalTranscriptionJob}
+transcribe_ListMedicalTranscriptionJobs:
+  title: List &TSCM; transcription jobs using an &AWS; SDK
+  title_abbrev: List medical transcription jobs
+  synopsis: list &TSCM; transcription jobs.
+  category:
+  languages:
+    JavaScript:
+      versions:
+        - sdk_version: 3
+          github: javascriptv3/example_code/transcribe
+          sdkguide: sdk-for-javascript/v3/developer-guide/transcribe-medical-examples-section.html#transcribe-list-medical-jobs
+          excerpts:
+            - description: Create the client.
+              snippet_tags:
+                - transcribe.JavaScript.createclientv3
+            - description: List medical transcription jobs.
+              snippet_tags:
+                - transcribe.JavaScript.jobs.createMedicalJobV3
```

Comment:
This excerpt calls StartMedicalTranscriptionJob, not ListMedicalTranscriptionJobs.

Reply:
fixed.